scripts/artifacts-building/containers/build-process.sh

```diff
@@ -77,6 +77,12 @@ patch_all_roles
 
 # Run playbooks
 cd /opt/rpc-openstack/openstack-ansible/playbooks
+
+# The host must only have the base Ubuntu repository configured.
+# All updates (security and otherwise) must come from the RPC-O apt artifacting.
+openstack-ansible ${RPCD_DIR}/playbooks/configure-apt-sources.yml -e "host_ubuntu_repo=http://mirror.rackspace.com/ubuntu"
```

Comment:
Oh yeah, we don't destroy the cache anymore like you used to require :p 
I'd modify the comment there to add something like:

```
# If we modify this now, the apt list will be ready for lxc_hosts,
# who will copy the apt sources from the host into the default
# container. This default container cache (built in
# lxc_hosts role) is then used in artifact building process.

```
Remember, not everyone has a deep knowledge of the roles in OSA and this avoid unnecessary loookup for ppl.